### ISSUE TITLE
Ways to add access control to reports

### ISSUE THREAD

**Issue by ctron** [trustification/trustify#334]
## nginx with basic auth

One easy way could be to use nginx with "Basic authentication": https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-http-basic-authentication/

## OpenShift OIDC proxy

The other one would be: https://github.com/openshift/oauth-proxy ... This would allow to re-use cluster users vs creating a custom `htpasswd` map.

The idea is to create a sidecar for the nginx pod. Expose the oidc proxy instead of directly nginx. and have the oidc proxy enforce access.

**Comment by bxf12315** [comment]
Is this issue just for trustify, or does it also need to be implemented on trustification?

**Comment by ctron** [comment]
You're right, this is the wrong repository. I'll try to transfer this issue.
